samples/sensor/bmp180/boards/nrf52840dongle_nrf52840.overlay

```diff
@@ -0,0 +1,26 @@
+// To get started, press Ctrl+Space to bring up the completion menu and view the available nodes.
+// For more help, browse the DeviceTree documentation at https://docs.zephyrproject.org/latest/guides/dts/index.html
+// To get started, press Ctrl+Space to bring up the completion menu and view the available nodes.
+// For more help, browse the DeviceTree documentation at https://docs.zephyrproject.org/latest/guides/dts/index.html
+/ {
+	chosen {
+		zephyr,console = &cdc_acm_uart0;
+	};
+};
+
+&zephyr_udc0 {
+	cdc_acm_uart0: cdc_acm_uart0 {
+		compatible = "zephyr,cdc-acm-uart";
+		label = "CDC_ACM_0";
+	};
+};
+&i2c0 {
+	sda-pin = < 13 >;
+	scl-pin = < 15 >;
```

Comment:
Please update to use pinctrl

Reply:
removed overlay for dongle, added for nrf52840dk instead